megablocks/layers/arguments.py

```diff
@@ -40,13 +40,15 @@ class Arguments:
     memory_optimized_mlp : bool = False
     mlp_type : str = 'mlp'
     grouped_mlp : bool = False
+    torch_mlp: bool = False
```

Comment:
Now that we're going to have 3 MLP implementations can we replace `grouped_mlp` and `torch_mlp` with an `mlp_impl` flag which can be "sparse", "grouped" or "torch"? i.e., like what we do with `mlp_type`?

Then you can tweak the dmlp_registry to take this into account.

Reply:
refactored